### PR TITLE
Add more links to API docs 

### PR DESCRIPTION
**Pull request by paulbalaji**
#### Description

- added more links to api docs
- did a couple of minor updates to world commands pages

(consider reading in commit order)

#### Tests

- improbadoc serve

#### Documentation

this is the documentation

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: docs/modules/player-lifecycle/heartbeating.md [hunk 1]
```diff
@@ -17,16 +17,16 @@ A server sends a heartbeat request at regular intervals to the clients. If the s
 
 ### Initialisation
 
-On the server-worker, the [`PlayerHeartbeatInitializationSystem`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerHeartbeat/PlayerHeartbeatInitializationSystem.cs) checks for ECS entities in the worker's view that have a `PlayerHeartbeatClient` component, but do not yet have the `HeartbeatData` component. For all these entities, a `HeartbeatData` component is added to the ECS entity at the end of the current frame.
+On the server-worker, the [`PlayerHeartbeatInitializationSystem`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerHeartbeat/PlayerHeartbeatInitializationSystem.cs) checks for ECS entities in the worker's view that have a `PlayerHeartbeatClient` component, but do not yet have the [`HeartbeatData`]({{urlRoot}}/api/player-lifecycle/heartbeat-data) component. For all these entities, a [`HeartbeatData`]({{urlRoot}}/api/player-lifecycle/heartbeat-data) component is added to the ECS entity at the end of the current frame.
```
Comment (jamiebrynes7) [inline docs/modules/player-lifecycle/heartbeating.md]: Why aren't we linking to the system as well?

File: docs/modules/player-lifecycle/heartbeating.md [hunk 2]
```diff
@@ -38,6 +38,6 @@ The client-worker runs the [`HandlePlayerHeartbeatRequestSystem`](https://github
 
 ### How to handle PlayerHeartbeat responses
 
-The [`HandlePlayerHeartbeatResponseSystem`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerHeartbeat/HandlePlayerHeartbeatResponseSystem.cs) on the server-worker iterates through all `PlayerHeartbeat` responses received. If at least one successful response was received from a player entity, the `NumFailedHeartbeats` inside this player entity's `HeartbeatData` component is set to 0.
+The [`HandlePlayerHeartbeatResponseSystem`](https://github.com/spatialos/gdk-for-unity/blob/master/workers/unity/Packages/com.improbable.gdk.playerlifecycle/Systems/PlayerHeartbeat/HandlePlayerHeartbeatResponseSystem.cs) on the server-worker iterates through all `PlayerHeartbeat` responses received. If at least one successful response was received from a player entity, the `NumFailedHeartbeats` inside this player entity's [`HeartbeatData`]({{urlRoot}}/api/player-lifecycle/heartbeat-data) component is set to 0.
```
Comment (jamiebrynes7) [inline docs/modules/player-lifecycle/heartbeating.md]: Similarly here

File: docs/modules/transform-sync/overview.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
+<%(TOC)%>
```
Comment (jamiebrynes7) [inline docs/modules/transform-sync/overview.md]: I don't think we need a ToC on this page

File: docs/reference/concepts/logging.md [hunk 4]
```diff
@@ -3,27 +3,27 @@
 #  Logs
  _This document relates to both the [MonoBehaviour and ECS workflows]({{urlRoot}}/reference/workflows/which-workflow)._
 
-The SpatialOS GDK for Unity uses a custom `ILogDispatcher` interface instead of `UnityEngine.Debug`, which gives more flexibility to handle logs separately in different workers and gives more context when handling the logs. There are two provided implementations of this interface:
+The SpatialOS GDK for Unity uses a custom [`ILogDispatcher`]({{urlRoot}}/api/core/i-log-dispatcher) interface instead of `UnityEngine.Debug`, which gives more flexibility to handle logs separately in different workers and gives more context when handling the logs. There are two provided implementations of this interface:
```
Comment (jamiebrynes7) [inline docs/reference/concepts/logging.md]: I like how we talk about the `ILogDispatcher` as if we didn't write this as the logging "framework" 😂

File: docs/reference/workflows/ecs/interaction/commands/world-commands.md [hunk 5]
```diff
@@ -1,115 +1,90 @@
 <%(TOC)%>
+
 # ECS: World commands
- _This document relates to the [ECS workflow]({{urlRoot}}/reference/workflows/which-workflow)._
 
 World commands are special commands that are sent to the SpatialOS runtime to ask it to reserve entity ids, create or delete entities, or request information about entities. (See the SpatialOS documentation on [world commands](https://docs.improbable.io/reference/latest/shared/design/commands#world-commands) for more information.)
 
-Each ECS entity that represents a SpatialOS entity has a set of components for sending world commands. For each world command, there is a component to send the command and receive the response.
+World command requests can be created through the `CommandSystem`, which also handles received responses.
```
Comment (jamiebrynes7) [inline docs/reference/workflows/ecs/interaction/commands/world-commands.md]:
```suggestion
World command requests can be sent with the `CommandSystem`, which also handles received responses.
```

File: docs/reference/workflows/monobehaviour/interaction/commands/world-commands.md [hunk 6]
```diff
@@ -22,45 +22,30 @@ There are two types of commands in SpatialOS:
 
 The commands documentation is:
 
-* [MonoBehaviour world commands]({{urlRoot}}/reference/workflows/monobehaviour/interaction/commands/world-commands)
+* MonoBehaviour world commands - this document
 * [ECS world commands]({{urlRoot}}/reference/workflows/ecs/interaction/commands/world-commands)
 * [MonoBehaviour component commands]({{urlRoot}}/reference/workflows/monobehaviour/interaction/commands/component-commands)
-* ECS component commands - this document
+* [ECS component commands]({{urlRoot}}/reference/workflows/ecs/interaction/commands/component-commands)
 * Both workflows - [world commands API reference]({{urlRoot}}/api/core/commands/world-commands)
```
Comment (jamiebrynes7) [inline docs/reference/workflows/monobehaviour/interaction/commands/world-commands.md]: Can we actually remove this little section? I feel like its not very useful and points you all over the shop.

File: docs/modules/player-lifecycle/heartbeating.md (hunk 1 above)
Comment (paulbalaji) [reply docs/modules/player-lifecycle/heartbeating.md]: missed it, gonna fix

File: docs/modules/player-lifecycle/heartbeating.md (hunk 2 above)
Comment (paulbalaji) [reply docs/modules/player-lifecycle/heartbeating.md]: missed it, gonna fix

File: docs/modules/transform-sync/overview.md (hunk 3 above)
Comment (paulbalaji) [reply docs/modules/transform-sync/overview.md]: trying to be consistent with a couple of other modules - but probably makes sense to have no ToC on all of the module overview pages

**Pull request merged**